float64 new_theta
float64 new_x
float64 new_y
---
